bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/profile/ScriptProfile.java

```diff
@@ -92,18 +105,25 @@ public ProfileTypeUID getProfileTypeUID() {
 
     @Override
     public void onStateUpdateFromItem(State state) {
+        if (isConfigured && !toHandlerStateScript.isBlank()) {
```

Comment:
Is it necessary to check `!toHandlerStateScript.isBlank()` here? `executeScript` checks `.isBlank()` anyway.

Reply:
I haven't noticed that, it is not neccassary to double check.